packages/visual-editor/src/components/RootRenderer/RootRenderer.tsx

```diff
@@ -123,7 +123,12 @@ export const RootRenderer: React.FC<RootRendererProperties> = ({ onChange }) =>
       <div data-ctfl-root className={styles.container} ref={containerRef} style={containerStyles}>
         {userIsDragging && <div data-ctfl-zone-id={ROOT_ID} className={styles.hitbox} />}
         <Dropzone zoneId={ROOT_ID} resolveDesignValue={resolveDesignValue} />
-        {userIsDragging && <div data-ctfl-zone-id={ROOT_ID} className={styles.hitboxLower} />}
+        {userIsDragging && (
+          <>
```

Comment:
I think a comment here would be good to explain why this seemingly empty div tag is here